be/src/vec/data_types/convert_field_to_type.cpp

```diff
@@ -257,7 +258,8 @@ void convert_field_to_typeImpl(const Field& src, const IDataType& type,
         JsonbWriter writer;
         Field::dispatch([&writer](const auto& value) { FieldVisitorToJsonb()(value, &writer); },
                         src);
-        *to = JsonbField(writer.getOutput()->getBuffer(), writer.getOutput()->getSize());
+        *to = JsonbField(writer.getOutput()->getBuffer(),
+                         static_cast<UInt32>(writer.getOutput()->getSize()));
```

Comment:
why use `static_cast` not `cast_set`, `cast_set` can check overflow in DEBUG MODE

Reply:
1. Actually the impl of `cast_set` will do value check in all build type.
2. `convert_field_to_typeImpl` will be called on every tuple of the array. Don't want to introduce performance issue so I tried `cast_set<UInt32,UInt64, false>`
3. After I checked the impl of `cast_set`, I found `cast_set<UInt32,UInt64, false>` equals to static_cast so I use static_cast